### ISSUE TITLE
Issue while using shortcut -c for config path

### ISSUE THREAD

**Issue by Jibbarth** [nunomaduro/phpinsights#123]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| Library version  | 1.3.1

I encouter an exception when using shortcut `-c` for loading a custom config path.

How to reproduce (on this repo): 

```
$ php bin/phpinsights analyse -c stubs/config.php
```
throw following exception 

```
In DelegatingLoader.php line 37:
                                                                             
  [Symfony\Component\Config\Exception\LoaderLoadException]                   
  Cannot load resource "/my/awesome/path/phpinsights/stubs/config.php".                                                                            
```

After investigating, I found that it's `EcsContainer::make();` in fault in `InsightFactory`.

In effect, when EscContainer is build, it will looking for detectFromInput it's own config from ArgvInput (cf https://github.com/Symplify/EasyCodingStandard/blob/v5.4.16/bin/container.php#L14) by searching `--config` or `-c` argument.

I don't know if it should be prevent here by emptying `$_SERVER['argv']` before calling `EscContainer::make()` or if it should be fixed in https://github.com/Symplify/EasyCodingStandard :thinking:  

WDYT ? 

Maybe @TomasVotruba has an idea on this ?

**Comment by TomasVotruba** [comment]
Hi, I'm not sure what the issue is, but the container should be created only by new Kernel:
https://github.com/Symplify/EasyCodingStandard/blob/v5.4.16/bin/container.php#L37

**Comment by Jibbarth** [comment]
Thanks for your help @TomasVotruba :+1:

**Comment by Jibbarth** [comment]
Fixed in #133
